nomad/node_endpoint_test.go

```diff
@@ -639,29 +805,25 @@ func TestClientEndpoint_Register_GetEvals(t *testing.T) {
 	}
 
 	// Transition it to down and then ready
-	node.Status = structs.NodeStatusDown
-	reg = &structs.NodeRegisterRequest{
-		Node:         node,
+	req := &structs.NodeUpdateStatusRequest{
+		NodeID:       node.ID,
+		Status:       structs.NodeStatusDown,
 		WriteRequest: structs.WriteRequest{Region: "global"},
 	}
-
-	// Fetch the response
-	if err := msgpackrpc.CallWithCodec(codec, "Node.Register", reg, &resp); err != nil {
+	if err := msgpackrpc.CallWithCodec(codec, "Node.UpdateStatus", req, &resp); err != nil {
```

Comment:
Changes like these were necessary because the `Node.Register` endpoint doesn't set the node status anymore.

Reply:
We do register them earlier in the test, it's that maybe the `UpdateStatus` method didn't exist before so the `Register` was being used to update the status?

Reply:
Oh, the register call was above the fold and I missed it somehow. :+1: